build.ps1

```diff
@@ -34,3 +48,31 @@ else {
     Invoke-Expression -Command  "dotnet run"
     if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode)  }
 }
+
+# Get runtime version
+$artifactsPath = "$baseDir\artifacts"
+$cli = Get-ChildItem -Path $artifactsPath -Include func.dll -Recurse | Select-Object -First 1
+$cliVersion = [System.Diagnostics.FileVersionInfo]::GetVersionInfo($cli).FileVersion
+
+# Generate MSI installers for Windows
+@('x64', 'x86') | ForEach-Object { 
```

Comment:
May want to add msi files to the CLI release pipeline @yojagad has created, so the MSI files are also automatically uploaded to each release / tag we create on github.

Reply:
Added.